models/stg_klaviyo__event.sql

```diff
@@ -22,18 +22,28 @@ fields as (
 final as (
     
     select 
-        _variation as variation_id,
+        _variation as variation_id,  --Should we request this be added to the ERD?
         campaign_id,
         timestamp as occurred_at,
-        -- don't include datetime...
+        -- don't include datetime... --Should this comment be removed?
         flow_id,
         flow_message_id,
         id as event_id,
         metric_id,
         person_id,
         type,
         uuid,
-        property_value as numeric_value
+        property_value as numeric_value 
+        /*******************
+        Since `property_value` is a custom field we should probably add wording in the README 
+        that this field is included by default and will not need to be added within the variable.
+
+        Additionally, should we maybe consider adding something to the README that this field
+        is required in order for the package, and downstream models, to effectively generate the
+        final models? Finally, is there Klaviyo documentation stating that `property_value` is always
+        the field which will contain the numeric value associated with the event 
+        (ie the dollars associated with a purchase)?
+        *******************/
```

Comment:
ah yeah so `property_value` is actually a default property for Klaviyo events ([docs](https://help.klaviyo.com/hc/en-us/articles/115000751052-Track-and-Identify-API-Reference#reserved-event-properties11)), but Fivetran is erroneously treating it as a custom field and prefixing it with `property_`

Reply:
Great docs reference, but strange we prefix it with `property_`? Thanks for this clarification!